concent_api/utils/message_handler.py

```diff
@@ -0,0 +1,72 @@
+from functools                      import wraps
```

Comment:
You can put those two decorators in a single file `utils\decorators.py`. It would be more in python/django convention and less repetitive on imports.